docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[alert-schema]]
+
+== Alert schema
+
+{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices.  In 7.x versions, the index pattern was `.siem-signals-<spacename>`.  In 8.x versions, the index pattern is `.alerts-security.alerts-<spacename>`, and the field names have changed. The following table includes the new names and provides a cross-reference to the previous field names.
```

Comment:
It seems a little weird to lead with the 7.x info everywhere on this page, since these are the 8.0 docs. I know that a lot of customers will be upgrading from 7.x, but it could be distracting for new customers starting on 8.x. Also, why don't the 8.x field names have links like a lot the 7.x fields? Again, that seems to privilege 7.x.

This suggestion reorders to focus on 8.x first, and also specifies that _some_ fields are different, not all.

```suggestion
{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices. In 8.x versions, the index pattern is `.alerts-security.alerts-<spacename>`. In 7.x versions, the index pattern was `.siem-signals-<spacename>`, and some field names were different. The following table includes the current names and cross-references the legacy field names.
```